src/main/java/com/denizenscript/depenizen/bukkit/events/residence/PlayerCreatesResidenceScriptEvent.java

```diff
@@ -0,0 +1,58 @@
+package com.denizenscript.depenizen.bukkit.events.residence;
+
+import com.bekvon.bukkit.residence.event.ResidenceCreationEvent;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.depenizen.bukkit.objects.residence.ResidenceTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class PlayerCreatesResidenceScriptEvent extends BukkitScriptEvent implements Listener {
```

Comment:
`ResidenceCreationEvent` seems to be cancellable, should include that in the meta (and the same for other cancellable events, if any).